### ISSUE TITLE
ELBv2 attributes do not configure properly on create

### ISSUE THREAD

**Issue by appilon** [terraform-providers/terraform-provider-aws#3848]
Load Balancer attributes do not appear to be setting their values correctly on create.

### Terraform Version
```
Terraform v0.11.2
+ provider.aws v1.11.0
```

### Affected Resource(s)
- aws_lb

### Terraform Configuration Files
```hcl
resource "aws_lb" "lb_test" {
  name            = "test"
  internal        = true
  security_groups = ["${aws_security_group.alb_test.id}"]
  subnets         = ["${aws_subnet.alb_test.*.id}"]
  
  idle_timeout = 30
  enable_deletion_protection = false

  enable_http2 = false

  tags {
    Name = "TestAccAWSALB_basic"
  }
}

variable "subnets" {
  default = ["10.0.1.0/24", "10.0.2.0/24"]
  type    = "list"
}

data "aws_availability_zones" "available" {}

resource "aws_vpc" "alb_test" {
  cidr_block = "10.0.0.0/16"

  tags {
    Name = "terraform-testacc-lb-basic"
  }
}

resource "aws_subnet" "alb_test" {
  count                   = 2
  vpc_id                  = "${aws_vpc.alb_test.id}"
  cidr_block              = "${element(var.subnets, count.index)}"
  map_public_ip_on_launch = true
  availability_zone       = "${element(data.aws_availability_zones.available.names, count.index)}"

  tags {
    Name = "tf-acc-lb-basic-${count.index}"
  }
}

resource "aws_security_group" "alb_test" {
  name        = "allow_all_alb_test"
  description = "Used for ALB Testing"
  vpc_id      = "${aws_vpc.alb_test.id}"

  ingress {
    from_port   = 0
    to_port     = 0
    protocol    = "-1"
    cidr_blocks = ["0.0.0.0/0"]
  }

  egress {
    from_port   = 0
    to_port     = 0
    protocol    = "-1"
    cidr_blocks = ["0.0.0.0/0"]
  }

  tags {
    Name = "TestAccAWSALB_basic"
  }
}
```

### Expected Behavior
Application load balancer created with HTTP/2 disabled

### Actual Behavior
Application load balancer has HTTP/2 enabled

### Steps to Reproduce
1. `terraform apply`

### References
- https://github.com/terraform-providers/terraform-provider-aws/issues/3511#issuecomment-373338468

**Comment by mpilar** [comment]
I will look into this as soon as I have a chance. It's curious because the acceptance tests should be grabbing this.

**Comment by appilon** [comment]
@mpilar I'm actually already working on a fix. I know what the issue is feel I would love a review from you when my PR goes up

**Comment by bflad** [comment]
This has been released in [version 1.13.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1130-march-28-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
